wikiman.yaml

```diff
@@ -72,6 +72,7 @@ extensions:
     repoName: wikimedia/mediawiki-extensions-Kartographer
   - name: Math
     repoName: wikimedia/mediawiki-extensions-Math
+    repoRef: 2822d35380c87960c56111be025fd2c5817e919e
```

Comment:
I read your comment and the ticket but TBH I failed to understand why we need to pin back to this version; would we be better to either add a patch or actually backport the change to the repo upstream?

Reply:
Solely to prioritize the deployment of the security fix, that cleanup can happen in a follow up PR IMO

Reply:
using up-do-date code resolved this